easybuild/easyconfigs/d/DCA++/DCA++-CMake-CrayGNU-2015.11-cpu.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'Bundle'
+
+name = 'DCA++'
+version = '1.0'
+versionsuffix = '-cpu'
+
+#consider building everything with eb:
+#preinstallopts: change build_directory
+#disable clean build directory
+
+homepage = 'http://www.itp.phys.ethz.ch/research/comp/computation.html'
+description = """The DCA++ software project is a C++ implementation of the dynamical cluster approximation (DCA) and its DCA+ extension. It aims to solve lattice models of strongly correlated electron systems. This module bundles all the dependencies and does not make use of the GPU."""
```

Comment:
I would change the last sentence to "This module bundles all the dependencies for the CPU-only version."